### ISSUE TITLE
community.grainfather.com

### ISSUE THREAD

**Issue by tromaxland** [linjmeyer/tilt-pitch#20]
Hi, can tilt-pitch communicate with grainfather.com?

**Comment by linjmeyer** [comment]
Hi @tromaxland!   In theory, Pitch was developed so it can communicate with any 3rd party system.  Unfortunately though I can't seem to find any information on Granfather's API (or if they even have one) which means it would have nowhere to send data to.  I did find this thread that says it's on the road map but I'm not sure how official it is: https://grainfather.nolt.io/69

**Comment by tromaxland** [comment]
There is certainly something because my original tilt-pi software can report to grainfather.com.  I opened a case at grainfather.com to obtain info about their API.  Merci !

**Comment by linjmeyer** [comment]
@tromaxland thanks for the info, I was searching for docs using Google.  Knowing they have some level of support I created an account and was able to find enough info to implement it once logged in.

I just released it as v0.8.0 if you'd like to give it a shot, feedback is welcome.

Info for setting it up:
* Setting up the device in Grainfather https://github.com/linjmeyer/tilt-pitch#grainfather
* Example pitch.json config https://github.com/linjmeyer/tilt-pitch/blob/master/examples/grainfather/pitch.json
